### DISCUSSION TITLE
Organization owners do not need to assign Seats

### DISCUSSION THREAD

**Discussion by Khavdd** [community/community#143501] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Everyone,

I have a question to ask as follows:

I purchased the 14 seat GitHub Team Plan via email [it@trithucgroup.com] and this email counts as 1 seat. Now I want this email only to manage licenses/memberships, renewals... not participate in github activities just to be the owner of the organization without being charged (Similar to role Global Admin role, no need to assign a license on the Microsoft 365 portal). Is that Okay?

Or is there a way for me to transfer the owner (it@trithucgroup.com) to another email account? I want to not count 1 seat on the owner account, what do I need to do?

Thank you,

**Comment by thomassiard** [comment]
In GitHub's Team Plan, each "seat" is counted based on users who are active members of the organization, which includes owners. Currently, there's no role similar to a "Global Admin" as in Microsoft 365 that can manage settings without consuming a seat.

However, here are a couple of options that may help you achieve what you're looking for:

Transfer Ownership: You could transfer ownership to another user and then remove it@trithucgroup.com from the organization to free up that seat. To do this:

Go to your organization’s settings, select People, and then Manage Owners.
Assign a new owner and remove it@trithucgroup.com from the organization.
Use a Personal or Non-Licensed Email: Alternatively, you could set up a new email solely for GitHub Team management (e.g., a personal account that doesn’t need a paid seat) and add it as an owner. After that, remove the it@trithucgroup.com account.

Contact GitHub Support: If these solutions don’t quite meet your needs, GitHub Support may be able to provide guidance or consider adding a "billing-only" role in future updates if this is a common request.

**Reply by Khavdd** [reply]
Hi Thomassiard,

Thank you for your answer and detailed instructions. I need to confirm the following:
- Assign a new owner and remove [it@trithucgroup.com] from the organization. => With this method, the new Owner is still counted as 1 seat, right?
- Use a Personal or Non-Licensed Email: Alternatively, you could set up a new email solely for GitHub Team management (e.g., a personal account that doesn’t need a paid seat) and add it as an owner. After that, remove the [it@trithucgroup.com] account. => Is it the same as when I transfer a new Owner using a personal email? For example: I transfer it@gmail.com and give the owner permissions to this account > So this personal account is not counted as 1 seat?

Thank you,
